### ISSUE TITLE
Add new sponsors/partners to site

### ISSUE THREAD

**Issue by alicetragedy** [rails-girls-summer-of-code/summer-of-code#481]
This is a placeholder issue to keep track of the sponsors who still need to be added to the website (i.e. they've expressed interested or bought a package but we're still waiting for logo/blurb)

- [x] Fedora Project
- [x] Thoughtworks
- [x] GitLab
- [x] Bendyworks
- [x] Runtastic
- [x] RubyMotion
- [x] Basecamp
- [x] Wooga
- [x] Cobot
- [x] Articulate
- [x] Contenful
- [x] Honeybadger
- [x] NextCloud
- [x] BitCrowd
- [x] innoq
- [x] Synack
- [x] Sentry
- [x] openSUSE
- [x] Softwire
- [x] Buffer
- [x] Heroku
- [x] Mozilla
- [x] Mapbox
- [x] Electra
- [x] AWS

**Comment by bexelbie** [comment]
@alicetragedy Can you help me answer the question from our designer in https://pagure.io/design/issue/585 ? Specifically, they need to know if you want a white background or transparent image.  Also, can you share where the image will appear?

**Comment by alicetragedy** [comment]
Hi @bexelbie — sure thing! Sponsors will appear both on the [sponsors page](https://railsgirlssummerofcode.org/sponsors/) and in our footer (see [our main page](https://railsgirlssummerofcode.org/) for an example). In both cases, they'll appear on a white background.
The transparent-background logo they've shared actually looks great, so if they are happy with that one, I am, too!

**Comment by bexelbie** [comment]
@alicetragedy The transparent one is good to go :)  Would you like me to do anything special to get it to you?

**Comment by alicetragedy** [comment]
@bexelbie no, it looks like I can just download it (I just tried and it worked). I'll add this to the website very soon — will still need a text blurb whenever you can. please let me know if I help out with that somehow!

**Comment by bexelbie** [comment]
@alicetragedy you did it by reminding me here.  Is this ok?

> The Fedora Project is a community of people working together to build a free and open source software platform and to collaborate on and share user-focused solutions built on that platform. Or, in plain English, we make an operating system and we make it easy for you do useful stuff with it.

If you need me to mail it to somewhere special, let me know.

**Comment by alicetragedy** [comment]
@bexelbie ah! awesome. I'd taken part of this text as a placeholder from your site while waiting for the Real Thing™. I'll add this shortly to the site 🎉
